core/abbreviate/abbreviate.py

```diff
@@ -2,11 +2,11 @@
 
 from talon import Context, Module
 
+from ..user_settings import get_list_from_csv
+
 mod = Module()
 mod.list("abbreviation", desc="Common abbreviation")
 
-# TODO: Make this a csv file. Not necessarily a settings/ csv file, it might be
-# better to be like homophones.csv.
 abbreviations = {
```

Comment:
since create spoken forms imports this dictionary, I think this might cause problems as-is
https://github.com/knausj85/knausj_talon/blob/b11b773a46f4f299ddcce4f9f84a566e78d0bb14/core/create_spoken_forms.py#L9
namely, don't think it would be in sync with the users file as written?

Reply:
Thanks for the pickup! I think I'll expose a function from abbreviate.py that gives you the list and use that in create_spoken_forms.py.